lib/dialects/sqlite3/schema/internal/unparser.js

```diff
@@ -0,0 +1,19 @@
+function unparseCreateIndex(ast, wrap = (v) => v) {
```

Comment:
I would suggest to use term `compile` instead of `unparse`

Reply:
I couldn't think of a better name but I like that suggestion.